### PR TITLE
Footer

### LINKED ISSUE #11: Footer
image

### PR DESCRIPTION
**Pull request by tanishqsindhu**

### CODE REVIEW

**Review by arpitBhalla — CHANGES_REQUESTED** [review]
Make it responsive

**Comment by abhii30** [comment]
Group work by-
tanishqsindhu
abhii30
bhavyajain8

File: index.html [hunk 1]
```diff
@@ -1,12 +1,91 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
   <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (arpitBhalla) [inline index.html]: Don't remove this line

**Review by arpitBhalla — CHANGES_REQUESTED** [review]
Still not working on mobile

**Review by arpitBhalla — APPROVED** [review]

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import logo from "./logo-footer.png";
+import "./App.css";
```
Comment (arpitBhalla) [inline src/App.js]: Use CSS modules, as discussed in meeting.

**Review by arpitBhalla — CHANGES_REQUESTED** [review]

File: src/App.js [hunk 3]
```diff
@@ -1,102 +1,13 @@
-import React from "react";
-import logo from "./logo-footer.png";
-import "./App.css";
+import './App.css';
```
Comment (arpitBhalla) [inline src/App.js]: Use CSS modules

**Review by arpitBhalla — CHANGES_REQUESTED** [review]

File: src/Components/footer.module.css [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+h2:before{
+    content: '';
+    position: absolute;
+    bottom: -5px;
+    left: 0;
+    width: 50px;
+    height: 2px;
+    background: #f00;
+}
+p{
```
Comment (arpitBhalla) [inline src/Components/footer.module.css]: Can you use classNames instead of element selectors.

**Review by arpitBhalla — CHANGES_REQUESTED** [review]

File: src/Components/footer.module.css (hunk 4 above)
Comment (tanishqsindhu) [reply src/Components/footer.module.css]: ok sir

**Review by arpitBhalla — APPROVED** [review]

File: src/components/Footer/index.js [hunk 5]
```diff
@@ -0,0 +1,140 @@
+import React from "react";
+import footer from "./footer.module.css";
+import logo from "./logo-footer.png";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faFacebookF,
+  faTwitter,
+  faInstagram,
+} from "@fortawesome/free-brands-svg-icons";
```
Comment (arpit-bhalla) [inline src/components/Footer/index.js]: Use SVG icons instead, https://simpleicons.org/

**Review by arpit-bhalla — CHANGES_REQUESTED** [review]

**Pull request closed without merging**